cardano-api/src/Cardano/Api/LedgerState.hs

```diff
@@ -77,6 +77,15 @@ module Cardano.Api.LedgerState
   -- ** Environment
   , Env(..)
   , genesisConfigToEnv
+
+  , ByronGenesisConfig
```

Comment:
Let's move these types to `Cardano.Api.Genesis` and export them via `Cardano.Api` instead.

Reply:
Done.  Some additional types that were defined in this module also had to move with them.